authorizations.md

```diff
@@ -0,0 +1,92 @@
+# Authorizations Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+An authorizations is the representation of some rights that are available to a specific user (eperson) in a defined scope, eventually the whole repository (site object).
+All the authorizations are always explicitly listed regardless to how they are grant, by direct policies via groups' membership or open to everyone (anonymous users).
+
+## Main Endpoint
+**/api/authz/authorizations**
+
+As we don't have yet an use case to iterate over all the existent authorizations the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Single Authorization
+**/api/authz/resourcepolicies/<:id>**
+
+Provide detailed information about a specific authorization. The JSON response document is as follow
+
+```json
+{
+  "id": [eperson_uuid-]feature_id-scope_uuid,
+  "type": "authorization"
+}
+```
+
+Attributes
+* id: the id of the authorization resource is defined by the combination of the eperson uuid (if not null), the feature id and the scope uuid joined with a minus 
+
+Exposed links:
+* eperson: link to the eperson that the authorization belong to. Can be null for authorizations grant to unlogged users
+* feature: link to the feature enabled by this authorization
+* scope: link to the object scope where this authorization apply. Not limited to DSpace objects, see [features endpoint](features.md) for more details
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated and the policy is not related to the ANONYMOUS user (eperson null)
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators and the user mentioned in the authorization can access
+* 404 Not found - if the authorization doesn't exist (or was already deleted)
+
+### Search methods
+#### resource
+**/api/authz/authorizations/search/scope?uri=<:uri>[&eperson=<:uuid>&feature=<:string>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uri: mandatory, the scope to use for the authorization check. The full URI of the rest resource must be specified, i.e. https://{dspace.url}/api/core/community/{uuid}
+* eperson: optional, the uuid of the eperson to evaluate for authorization. If not specified authorization of anonymous users will be returned
```

Comment:
Why is this required. I would expect the current user to be used for verifying the user's authorizations

Reply:
this would allow an administrator to inspect witch permissions has a different user. If we found difficult to implement we can always reduce the first version to "only the current user" but, also if we do that, keeping this as parameter will allow us to evolve without changing the contract.
Anyway, we can and should revisit that after some concrete experiment in the implementation